### DISCUSSION TITLE
Joomla 4 - No “Edit” submenus to “Actions” on selected menu item ?

### DISCUSSION THREAD

**Discussion by Chacapamac** [joomla/joomla-cms#35317] [General] [answered]
When you want to edit a menu item you need to click on its name/link, great. If you select (tick) a menu item and go to the “Actions” button, you're out of luck. You don't have the option to edit.

If you can select an element and have “ an “Actions” button you should have All the possible actions in the drop-down and no “Actions” should be missing.

This is a UI mistake.
![menu-item-edition](https://user-images.githubusercontent.com/1907276/130453282-9bc9aeb0-25e7-44c4-a1db-3d2593543a5e.jpg)

**Comment by PhilETaylor** [comment]
Someone else mentioned this yesterday (Edit: Ah it was you :) https://github.com/joomla/joomla-cms/issues/35301#issuecomment-903308478) 

I'm sure @brianteeman had a reason he gave for this no longer being there a long time ago

It's by design but I can't remember why

**Comment by brianteeman** [comment, marked as answer]
There are two reasons I can think of

1. Previously you could check multiple articles and then click edit but it would only let you edit one article.
2. Why would someone click to check, scroll up, click to expand actions, click to edit. When they can just click once on the link.

**Reply by simbus82** [reply]
> There are two reasons I can think of
> 
> 1. Previously you could check multiple articles and then click edit but it would only let you edit one article.

"Edit" can be shown only if one article is selected

> 2. Why would someone click to check, scroll up, click to expand actions, click to edit. When they can just click once on the link.

Why? Because they can!
This is a basic principle of UX. If you build an interface the a user can use, you have to allow the user to use it in the best way possible. In this example like an alternative way or as a fallback for the inexperienced user.

There is also a logical aspect, which is what @Chacapamac noted.
If you give me the possibility to select only one item and you light up the "Actions" button, it's OBVIOUS that under "actions" I expect  the presence of every possible action that I can do on the selected item.

**Reply by brianteeman** [reply]
Don't see the need myself but pull requests always welcome

**Comment by Chacapamac** [comment]
> Previously you could check multiple articles and then click edit but it would only let you edit one article.

**Good enough for me, make complete sense!**

> Why would someone click to check, scroll up, click to expand actions, click to edit. When they can just click once on the link.

Somebody like me. Working on a 27 inches Imac with my mouse at the highest tracking speed. I did have problem to click on the menu item link, it was inconsistent, sometime opening the edit tab, sometimes nothing. I even open a Github Issue on this (closed now).

This is for that reason that I instinctively select the menu item and look for the “Edit” function in the “Functions”.

The problem was the tracking speed, and the possibility of ordering item together that didn’t register my click as a direct click/select when I was clicking. When I put the mouse cursor carefully on the link without moving, the link work at all time.

Just hope not to much people experience the same thing. To fast on the draw!
